pages/howWeCalculate.js

```diff
@@ -50,21 +49,21 @@ export default function howWeCalculate() {
             methods to 20% or more.
           </TextBlock>
           <TextBlock title="How does it work?" anchor="howItWorks">
-            The calculator uses inputs from the staff survey, taking into
-            account commute distance and transport modes. It plugs those numbers
-            into this formula from the{" "}
-            <Link
-              color="#044B7F"
-              textDecoration="underline"
-              textUnderlineOffset="3px"
-              fontWeight={700}
-              href="https://ghgprotocol.org"
-            >
-              Greenhouse Gas Protocol
-            </Link>{" "}
-            that calculates the CO2e emissions from employee travel. The results
-            report provides details on the calculations and emissions factors
-            used.
+            We use the responses collected from the two week-long survey on how
+            employees travelled to work to estimate how much CO2 is emitted by
+            employees’ commutes in one typical work week.
+            <br />
+            <br />
```

Comment:
Is there any rule regarding two `<br />` tags one after another? Looks a bit unusual to see it used for adding an additional space rather than just a line break. What do you think?

Reply:
Yea, I get you @OlaShabalina 😅 

I used 2 `<br />`s to get the spacing between paragraphs looking like the figma design.

I initially had them set side by side (`<br /><br />`), but `prettier`, in its infinite wisdom, thought it better to have them on separate lines 🤷🏽 

How would you have preferred to see it done?

Reply:
Let's merge for now and decide if we want to change it later!